File: CS/CS/WinWebSolution.Module.Win/CustomUserControls/WinCustomUserControl.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Drawing;
using System.Data;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraEditors;

namespace WinCustomUserControlLibrary {
    public partial class WinCustomUserControl : DevExpress.XtraEditors.XtraUserControl {
        public WinCustomUserControl() {
            InitializeComponent();
        }
        protected override void OnLoad(EventArgs e) {
            base.OnLoad(e);
            gridControl1.DataSource = Demo.DataHelper.GetData();
        }
    }
}

File: VB/VB/WinWebSolution.Module.Win/CustomUserControls/WinCustomUserControl.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Drawing
Imports System.Data
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraEditors

Namespace WinCustomUserControlLibrary
	Partial Public Class WinCustomUserControl
		Inherits DevExpress.XtraEditors.XtraUserControl
		Public Sub New()
			InitializeComponent()
		End Sub
		Protected Overrides Sub OnLoad(ByVal e As EventArgs)
			MyBase.OnLoad(e)
			gridControl1.DataSource = Demo.DataHelper.GetData()
		End Sub
	End Class
End Namespace
